examples/docs_snippets/docs_snippets/concepts/assets/load_asset_values.py

```diff
@@ -40,25 +40,21 @@ def asset2():
 def load_single_asset_value():
     # single_asset_start_marker
 
-    @repository
-    def repo():
-        return [load_assets_from_current_module()]
+    defs = Definitions(assets=[load_assets_from_current_module()])
```

Comment:
(github doesn't allow me to suggest on unchanged lines)
```
# single_asset_start_marker

defs = Definitions(assets=[load_assets_from_current_module()])
```
=>
```
@asset
def asset1():
    pass

# single_asset_start_marker
defs = Definitions(assets=[asset1])
```

looks like the error says that `load_assets_from_current_module()` is empty so instead of loading from current module, we can load an asset1 in this case

Reply:
This change wound up producing this error:

```
>       defs = Definitions(assets=asset1)
E       NameError: name 'asset1' is not defined

../docs_snippets/concepts/assets/load_asset_values.py:39: NameError
```

But this gets the tests to pass:

```python
# single_asset_start_marker
defs = Definitions(assets=assets)
```

I moved the example start marker down to exclude this from the example, as I thought it might be confusing. It also makes the single and multiple value examples consistent. I'm not sure if this is the best approach though, so open to feedback.